src/views/BookList.vue

```diff
@@ -0,0 +1,45 @@
+<template lang="pug">
+  div.book-list-container
+    book(v-for="book in books" :key="book.id" :book="book")
+</template>
+
+<script>
+import { getBooks } from '@/services/BookService'
+import Book from '@/components/Book'
+
+export default {
+  name: 'BookList',
+  components: {
+    Book
+  },
+  data () {
+    return {
+      books: []
+    }
+  },
+  methods: {
+    async fetchBooks () {
+      const response = await getBooks()
+      if (response.ok) {
+        this.books = response.data
+      } else {
+        this.books = [{ id: 1, title: 'Título del libro', author: 'Autor del libro', img: null }, { id: 1, title: 'Un titlo', author: 'Un autor', img: null }, { id: 1, title: 'Un titlo', author: 'Un autor', img: null }, { id: 1, title: 'Un titlo', author: 'Un autor', img: null }, { id: 1, title: 'Un titlo', author: 'Un autor', img: null }, { id: 1, title: 'Un titlo', author: 'Un autor', img: null }, { id: 1, title: 'Un titlo', author: 'Un autor', img: null }, { id: 1, title: 'Un titlo', author: 'Un autor', img: null }, { id: 1, title: 'Un titlo', author: 'Un autor', img: null }]
```

Comment:
move this  to a constant file

Reply:
It's kind of temporary... because the API doesn't work. But fine.